css/www/components/page.css

```diff
@@ -61,6 +61,11 @@ body .www-component-anchors .www-component-anchor[option][selected] {
   margin-bottom: var(--space-6);
 }
 
+.www-component-desc a {
+  text-decoration: none;
+  color: var(--color-font-primary:);
```

Comment:
i think this should not be generic. 

I made a similar change here 
https://github.com/ampproject/ampstart/blob/master/css/www/howitworks/page.css#L25

May be we could make a class i shared.css - www-anchor-highlight (or a bette name)? and reuse across?